src/app/clusters/scenes-server/SceneTableImpl.cpp

```diff
@@ -737,11 +737,17 @@ CHIP_ERROR DefaultSceneTableImpl::SceneSaveEFS(SceneTableEntry & scene)
 {
     if (!HandlerListEmpty())
     {
-        uint8_t clusterCount = 0;
-        clusterId cArray[kMaxClustersPerScene];
-        Span<clusterId> cSpan(cArray);
-        clusterCount = GetClustersFromEndpoint(cArray, kMaxClustersPerScene);
-        cSpan.reduce_size(clusterCount);
+        uint8_t clusterCount = GetClusterCountFromEndpoint();
```

Comment:
So this will include client clusters.  That seems like it will over-allocate...  Maybe it's OK for now, but perhaps a followup to ask for the number of _server_ clusters?

Reply:
issue opened [here](https://github.com/project-chip/connectedhomeip/issues/24177)